### PR TITLE
Change and update

### PR DESCRIPTION
**Pull request by saderi**
Hi,
I made some changes in compose file and add description to README.md
I hope it would be useful :)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
-    links:
-      - postgres:postgres
-    image: fenglc/pgadmin4
+    container_name: pgadmin_container
+    image: dpage/pgadmin4
+    environment:
+      PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL:-pgadmin4@pgadmin.org}
+      PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
     volumes:
-       - /data/pgadmin:/root/.pgadmin
```
Comment (khezen) [inline docker-compose.yml]: keep /data

File: docker-compose.yml [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
+       - pgadmin:/root/.pgadmin
     ports:
-      - "5050:5050"
+      - "${PGADMIN_PORT:-5060}:80"
```
Comment (khezen) [inline docker-compose.yml]: ${PGADMIN_PORT:-5050}:80

File: README.md [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
-
-# setup
-
-## steps
-1. `git clone https://github.com/Khezen/compose-postgres.git`
-2. `cd compose-postgres`
-3. `docker-compose up`
-
-## result
-* postgres running on 5432
```
Comment (khezen) [inline README.md]: keep postgres credentials

File: README.md [hunk 4]
```diff
@@ -1,18 +1,27 @@
...
+
+* `POSTGRES_USER` the default value is **postgres**
+* `POSTGRES_PASSWORD` the default value is **changeme**
+* `PGADMIN_PORT` the default value is **5060**
+* `PGADMIN_DEFAULT_EMAIL` the default value is **pgadmin4@pgadmin.org**
+* `PGADMIN_DEFAULT_PASSWORD` the default value is **admin**
+
+
+## Access to PgAdmin: 
+* **URL:** `http://localhost:5060`
```
Comment (khezen) [inline README.md]: * **URL:** `http://localhost:5050`

File: README.md [hunk 5]
```diff
@@ -1,18 +1,27 @@
...
+* docker-compose
+
+## Quick Start
+* Clone or download this repository
+* Go inside of directory,  `cd compose-postgres`
+* Run this command `docker-compose up -d`
+
+
+## Environments
+This Compose file uses several environment variables which are easy to miss.
```
Comment (khezen) [inline README.md]: This Compose file contains the following environment variables:

File: README.md [hunk 6]
```diff
@@ -1,18 +1,27 @@
...
+This Compose file uses several environment variables which are easy to miss. 
+
+* `POSTGRES_USER` the default value is **postgres**
+* `POSTGRES_PASSWORD` the default value is **changeme**
+* `PGADMIN_PORT` the default value is **5060**
+* `PGADMIN_DEFAULT_EMAIL` the default value is **pgadmin4@pgadmin.org**
+* `PGADMIN_DEFAULT_PASSWORD` the default value is **admin**
+
+
+## Access to PgAdmin:
```
Comment (khezen) [inline README.md:28]: ## Access

* pgadmin4
  * `127.0.0.1:5050`
  * user: `pgadmin4@pgadmin.org` (default)
  * password: `admin` (default)
* postgres
  *  `127.0.0.1:5432`
  * user: `postgres` (default)
  * password: `changeme` (default)

**Review by khezen — CHANGES_REQUESTED** [review]
Hi,

I requested two changes in the readme and one change in the compose. 
What do you think?

**Comment by saderi** [comment]
I have just changed you suggestion

**Review by khezen — APPROVED** [review]

**Pull request merged**